### ISSUE TITLE
[Feature] Locations to render Toast notifications

### ISSUE THREAD

**Issue by ani-ali** [ever-co/ever-teams#2663]
@evereq Here is the list of toast that I know. Please verify and modify if needed

There is list you can show to toast with translated messages.
- Create Task
- Everywhere when user copy something like ( Copy link in kanban board + Profile Page copy link )
- Invite member to our team modal 
- Show toast when create  [Task status, Task Priorities, Task Sizes, Task Sizes, Task Labels] in ( Setting page )
 
Not sure about danger zone @evereq can you guide 
![image](https://github.com/ever-co/ever-teams/assets/105014311/dad6dd5e-9e47-4218-82fa-f9e7c3103a56)

Also we can use toast to render error messages if something not does well like ( wrong password + wrong email ) etc

**Comment by evereq** [comment]
1. Create Task - we need to think how to visually highlight it if possible some other way, not with toast
2. Everywhere when user copy something - it should be tiny notification close to the area where user copied something (so not toast)
3. Invite member to our team modal - can be Toast or just inside page. I think we are doing now already inside of the page, no!? I asked someone to show screenshot of how we notify now to decide if we need to add toast or not
4. Show toast when create [Task status, Task Priorities, Task Sizes, Task Sizes, Task Labels] in ( Setting page ) - absolutely does not make any sense because for user s/he see right away that those new records are created in UI (and we can highlight that too inside page itself, e.g. show for few seconds some border around created item).

Basically all above does not make much sense to me, except possible 3 for which I am not sure what we are doing already now.

**Comment by evereq** [comment]
For Danger zone, I am not sure what displayed after those buttons are clicked, possible makes sense to show Toast that notify that "Team was removed" or "Ownership of XXX Team was transferred" or "You just quit Team XXX" etc, not sure, needs to see full flow as after some of those actions, user may logout maybe or switch to another team etc.
